openbas-api/src/main/java/io/openbas/migration/V3_35__Remove_cascade_delete_dependency.java

```diff
@@ -7,7 +7,7 @@
 import java.sql.Statement;
 
 @Component
-public class V3_34__Remove_cascade_delete_dependency extends BaseJavaMigration {
+public class V3_35__Remove_cascade_delete_dependency extends BaseJavaMigration {
```

Comment:
Your migration should have the number after the other one
You can't change migration number because it's already deploy on some DB